### DISCUSSION TITLE
部分神隐漫画存在问题

### DISCUSSION THREAD

**Discussion by lecromonda** [xiaoyaocz/flutter_dmzj#169] [General]
潘多拉之心，后街女孩的漫画打开后无法显示，似乎是加载失败
![F P1PK}N`IOLBEG$H}$~2PA](https://user-images.githubusercontent.com/80088580/234029588-00fb0886-fd76-4f2b-9173-8e9a8aefb06e.jpg)
![N2TL_7W}8$ITLITUX__8_5C](https://user-images.githubusercontent.com/80088580/234029611-2410c7ca-1eb6-4ca8-a2d5-cb56db70dec7.jpg)
![VMJ7O_H6BCPWYN1QBONA{_4](https://user-images.githubusercontent.com/80088580/234029633-f2f74e3b-03ec-4109-93b5-7aacc6d6d029.jpg)
![)(6LO ZLEW2DU 81V B 1KP](https://user-images.githubusercontent.com/80088580/234029647-988887a2-98e7-4e77-9d20-fd55c9492d1b.jpg)

**Comment by xiaoyaocz** [comment]
打开或关闭"优先加载高清图"选项试试

**Reply by lecromonda** [reply]
问题解决，也许可以把这个提示加入到app的开始页面里？

**Comment by lecromonda** [comment]
同时，在忍者杀手的漫画里无法正常显示所有内容，仅显示了第一卷，推测是没有完成分类，例如，单行本，其他汉化这种
![EDU}B04 POH$)(LVLRO%QVH](https://user-images.githubusercontent.com/80088580/234030322-a39066ca-266a-4709-9e44-041c61f2a679.jpg)

**Reply by xiaoyaocz** [reply]
使用V1接口获取的神隐漫画，只有连载/单行本两种分类。
人气及订阅为0时即为V1接口获取的神隐漫画

**Reply by lecromonda** [reply]
但忍杀可以直接在网页上打开，应该不属于神隐类吧

**Reply by xiaoyaocz** [reply]
APP有等级限制，你可以在官方APP上看看能不能看

**Reply by lecromonda** [reply]
lv0也可以正常打开漫画
![$M5@~$Q$X5)4MT4805S0M9](https://user-images.githubusercontent.com/80088580/234034618-5f1a855f-1f56-409e-8904-e396d9713861.jpg)
![HKME}3@NVS FH36HVK5M0`4](https://user-images.githubusercontent.com/80088580/234034629-c09bf38c-d499-4463-93c7-af0f20365cd2.jpg)
